lib/msal-browser/src/index.ts

```diff
@@ -133,6 +135,7 @@ export {
     // Server Response
     ExternalTokenResponse,
     // Utils
+    ProtocolUtils,
```

Comment:
This is an internal detail that should not be exported, let's discuss how you can get the functionality you need in another way. FetchClient too, though to a lesser extent given it implements a public interface